jwst/datamodels/fgs.py

```diff
@@ -0,0 +1,71 @@
+from __future__ import absolute_import, unicode_literals, division, print_function
+
+from . import model_base
+
+
+__all__ = ['FGSModel']
+
+
+class FGSModel(model_base.DataModel):
```

Comment:
For the 2 "raw" and "calibrated" versions of the data models, I would suggest trying not to use the somewhat generic "FGS" name, because the FGS can be used in science mode as well. These data models are specific to guide-mode data coming out of the FGS, so I think we should make that explicit in the data model names. And because the 2 data models are nearly identical to one another (the only difference is the shape of the SCI or "data" array), I would also suggest a common naming scheme for them, such as "GuiderRawModel" (or "GuiderRampModel", similar to our existing RampModel for raw science data) and "GuiderCalModel".